### PR TITLE
Added Danish translation

### PR DESCRIPTION
**Pull request by nikolajskov**

### CODE REVIEW

File: timeago/lib/src/messages/da_messages.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  String minutes(int minutes) => '$minutes minutter';
+  String aboutAnHour(int minutes) => 'omkring en time';
+  String hours(int hours) => '$hours timer';
+  String aDay(int hours) => 'en dag';
+  String days(int days) => '$days dage';
+  String aboutAMonth(int days) => 'omkring en måned';
+  String months(int months) => '$months måneder';
+  String aboutAYear(int year) => 'omkring et år';
+  String years(int years) => '$years år';
+  wordSeparator() => ' ';
```
Comment (andresaraujo) [inline timeago/lib/src/messages/da_messages.dart]: let's add `String` return value for `wordSeparator()`

File: timeago/lib/src/messages/da_messages.dart [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  String minutes(int minutes) => '$minutes min';
+  String aboutAnHour(int minutes) => '~1 t';
+  String hours(int hours) => '$hours t';
+  String aDay(int hours) => '~1 d';
+  String days(int days) => '$days d';
+  String aboutAMonth(int days) => '~1 md';
+  String months(int months) => '$months md';
+  String aboutAYear(int year) => '~1 år';
+  String years(int years) => '$years år';
+  wordSeparator() => ' ';
```
Comment (andresaraujo) [inline timeago/lib/src/messages/da_messages.dart]: let's add `String` return value for `wordSeparator()`

File: timeago/lib/src/messages/da_messages.dart [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  String aDay(int hours) => 'en dag';
+  String days(int days) => '$days dage';
+  String aboutAMonth(int days) => 'omkring en måned';
+  String months(int months) => '$months måneder';
+  String aboutAYear(int year) => 'omkring et år';
+  String years(int years) => '$years år';
+  wordSeparator() => ' ';
+}
+
+class EnShortMessages implements LookupMessages {
```
Comment (andresaraujo) [inline timeago/lib/src/messages/da_messages.dart]: The name for this should be `DaShortMessages`

**Comment by andresaraujo** [comment]
Just published 2.0.7 with your changes @nikolajskov 
Thanks!

**Comment by nikolajskov** [comment]
@andresaraujo You're welcome. Glad to help.

**Pull request merged**